README.md

```diff
@@ -63,41 +63,49 @@ matching route is found in the proxy table:
 ### Command-line options
 
 ```
-  Usage: configurable-http-proxy [options]
+Usage: configurable-http-proxy [options]
 
   Options:
 
     -h, --help                       output usage information
     -V, --version                    output the version number
     --ip <ip-address>                Public-facing IP of the proxy
     --port <n> (defaults to 8000)    Public-facing port of the proxy
-    
+
     --ssl-key <keyfile>              SSL key to use, if any
     --ssl-cert <certfile>            SSL certificate to use, if any
     --ssl-ca <ca-file>               SSL certificate authority, if any
     --ssl-request-cert               Request SSL certs to authenticate clients
     --ssl-reject-unauthorized        Reject unauthorized SSL connections (only meaningful if --ssl-request-cert is given)
-    --ssl-protocol <ssl-protocol>    Set specific HTTPS protocol, e.g. TLSv1_2, TLSv1, etc.
+    --ssl-protocol <ssl-protocol>    Set specific SSL protocol, e.g. TLSv1.2, SSLv3
     --ssl-ciphers <ciphers>          `:`-separated ssl cipher list. Default excludes RC4
     --ssl-allow-rc4                  Allow RC4 cipher for SSL (disabled by default)
     --ssl-dhparam <dhparam-file>     SSL Diffie-Helman Parameters pem file, if any
-    
+
     --api-ip <ip>                    Inward-facing IP for API requests
     --api-port <n>                   Inward-facing port for API requests (defaults to --port=value+1)
     --api-ssl-key <keyfile>          SSL key to use, if any, for API requests
     --api-ssl-cert <certfile>        SSL certificate to use, if any, for API requests
     --api-ssl-ca <ca-file>           SSL certificate authority, if any, for API requests
     --api-ssl-request-cert           Request SSL certs to authenticate clients for API requests
     --api-ssl-reject-unauthorized    Reject unauthorized SSL connections (only meaningful if --api-ssl-request-cert is given)
-    
+
     --default-target <host>          Default proxy target (proto://host[:port])
     --error-target <host>            Alternate server for handling proxy errors (proto://host[:port])
     --error-path <path>              Alternate server for handling proxy errors (proto://host[:port])
     --redirect-port <redirect-port>  Redirect HTTP requests on this port to the server on HTTPS
     --pid-file <pid-file>            Write our PID to a file
+
+    --storage-provider <provider>    The storage provider for the route table (defaults to memory)
```

Comment:
The reason I chose to use `--redis-XYZ` is to make it clear that these only apply to redis. I'm a little concerned that naming these options `--storage-provider-XYZ` would imply that there are multiple options for external storage.

I'm happy to concede here though, if you feel this would be better.